client/src/components/Ratings/Ratings.jsx

```diff
@@ -1,8 +1,85 @@
 import React, { useState, useEffect } from 'react';
+import ratingsAPI from '../../API/Ratings.js';
+import UserReviews from './ReviewCard.jsx';
```

Comment:
import ReviewCard from './ReviewCard.jsx';

Reply:
For some reason I have this on my machine already, so I'll push this again.